recipes/models.py

```diff
@@ -18,3 +20,12 @@ class Instruction(models.Model):
     ordinal = models.IntegerField(blank=True, null=True)
     instruction_text = models.TextField(null=True)
     recipe = models.ForeignKey(Recipe, related_name='instructions')
+
+class Profile(models.Model):
```

Comment:
I don't understand the point of the Profile class. Did you copy that from somewhere or come up with it yourself?

Reply:
It was from the sign up guide, but I think I might have gotten confused on the usage, because it doesn’t really look like it is doing anything. I can probably remove that part